contents.json

```diff
@@ -5352,10 +5357,10 @@
        "homepage": "https://github.com/vincent-pradeilles/weakable-self"
      },
     {
-       "title": "RequestPermission",
+       "title": "SPPermission",
```

Comment:
@ZulwiyozaPutra could you revert all the changes that don't regard to adding `Arale`?